docs/src/assets/scripts.js

```diff
@@ -178,7 +179,9 @@ const getWebpage = async (folder, hash, shallUpdateHistory) => {
     searchInput.focus();
     const searchResultsList = doc.getElementById('search-results-list');
     searchInput.addEventListener('keyup', async (evt) => {
-      const { hits } = await searchIndex.search(searchInput.value);
+      const { hits: searchHits } = await searchIndex.search(searchInput.value);
+      const { hits: discussionHits } = await discSearchIndex.search(searchInput.value);
+      const hits = [ ...searchHits, ...discussionHits ];
       if ( ! hits.length ) { return; }
       searchResultsList.innerHTML = '';
       hits.forEach((hit) => {
```

Comment:
You should add a new class for discussions and add some custom styling, like a Github Discussions logo